<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAUserDefineAxisDataWidget</class>
 <widget class="QWidget" name="SAUserDefineAxisDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>60</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>User Define Axis Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelStart">
       <property name="text">
        <string>start:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxStart">
       <property name="minimum">
        <double>-1000000000000000072531436381529235126158374409646521955518210155479040.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000004188152556421145795899143386664033828314342771180699648.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDetal">
       <property name="text">
        <string>increase:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxIncrease">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-10000000000000000102350670204085511496304388135324745728.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999999999995280522225138166806691251291352861698530421623488512.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelExample">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
